system-tests/dashboard/test-dashboard.js

```diff
@@ -0,0 +1,205 @@
+require("../_support/utils/ServicesUtil");
+const { Timeouts } = require("../_support/constants");
+
+// same parameters as defined in app.json
+const SERVICE_NAME = "dashboard-test-service";
+const SERVICE_CPUS = 0.1;
+const SERVICE_MEM = 10;
+const SERVICE_DISK = 10;
+
+function deleteService() {
```

Comment:
I think if we do this we want to make it more generic like passing filename as an argument for the app.json. And a name otherwise the app definition is to opinionated.